packages/office-addin-dev-certs/src/httpsServerOptions.ts

```diff
@@ -14,6 +15,12 @@ export async function getHttpsServerOptions(): Promise<IHttpsServerOptions> {
     await ensureCertificatesAreInstalled();
 
     const httpsServerOptions = {} as IHttpsServerOptions;
+    try {
+        httpsServerOptions.ca = fs.readFileSync(defaults.localhostCaCertificatePath);
+    } catch (err) {
+        throw new Error(`Unable to read the ca certificate file.\n${err}`);

```

Comment:
Make the `ca` uppercase. `Unable to read the CA certificate file.`  CA stands for Certificate Authority.